WalletWasabi/WebClients/ChaumianCoinJoin/WasabiClient.cs

```diff
@@ -21,6 +22,29 @@ public WasabiClient(Uri baseUri, IPEndPoint torSocks5EndPoint = null) : base(bas
 		{
 		}
 
+		#region blockchain
+
+		public async Task<IEnumerable<string>> GetFiltersAsync(string bestKnownBlockHash, int count)
```

Comment:
Can you do `Task<IEnumerable<FilterModel>> GetFiltersAsync(uint256 bestKnownBlockHash, int count)`?

Reply:
Yep.